src/wasm-traversal.h

```diff
@@ -50,6 +50,7 @@ struct Visitor {
   ReturnType visitLoad(Load* curr) {}
   ReturnType visitStore(Store* curr) {}
   ReturnType visitAtomicRMW(AtomicRMW* curr) {return ReturnType();} //Stub impl so not every pass has to implement this yet.
```

Comment:
Could we make all these functions have the return, I believe we decided that was best?

Also, please add spaces after `{` and before `}`.